client/src/pages/Login.js

```diff
@@ -150,127 +198,165 @@ const Login = ({ isLoginForm }) => {
-          <ButtonText>Gmail</ButtonText>
-        </SocialButton>
-        <SocialButton
-          style={StyleSocialIcon}
-          icon={<SvgIcon src={twitter} />}
-          onClick={() => handleSocialLogin("twitter")}
-        >
-          <ButtonText>Twitter</ButtonText>
-        </SocialButton>
-        <SocialButton
-          style={StyleSocialIcon}
-          icon={<SvgIcon src={linkedin} />}
-          onClick={() => handleSocialLogin("linkedin")}
-        >
-          <ButtonText>Linkedin</ButtonText>
-        </SocialButton>
-      </FlexBox>
-    </div>
+    <LoginContainer>
+       <LoginLeftContainer>
+           <SocialImageContainer>
+               <img
+                  src={isLoginForm ? socialmedia : socialmedia2}
+                  alt=""
+                  />
+           </SocialImageContainer>
+       </LoginLeftContainer>
+       <LoginRightContainer>
+          <div className="text-center">
+            <FormContainer>
+                <Title>{isLoginForm ? "Sign In" : "Sign Up"}</Title>
+                <form id="login-password" method="POST">
+                  <InputWrapper>
+                    <Label for="email" style={StyleLabel} label="E-mail" />
+                    <Input
+                      type="email"
+                      name="email"
+                      required
+                      placeholder="Enter email address"
+                      value={email}
+                      onChange={handleInputChangeEmail}
+                      style={StyleInput}
+                    />
+                  </InputWrapper>
+                  <InputWrapper>
+                    <Label for="password" style={StyleLabel} label="Password" />
+                    <Input
+                      type="password"
+                      name="password"
+                      id="password"
+                      required
+                      placeholder="Enter password"
+                      value={password}
+                      onChange={handleInputChangePassword}
+                      style={StyleInput}
+                    />
+                    <PasswordVisibility onClick={togglePasswordVisibility} src={eye} alt="" />
+                  </InputWrapper>
+                  {isLoginForm ? ''
+                   : ( <InputWrapper>
+                    <Label for="confirmPassword" style={StyleLabel} label="Confirm Password" />
+                    <Input
+                      type="password"
+                      name="confirmPassword"
+                      id="confirmPassword"
+                      required
+                      placeholder="Confirm password"
+                      onChange={handleInputChangeConfirmPassword}
+                      value={confirmPassword}
+                      style={StyleInput}
+                    />
+                    <PasswordVisibility onClick={toggleConfirmPasswordVisibility} src={eye} alt="" />
+                  </InputWrapper> ) }
+                  <SubmitButton
```

Comment:
Related to my previous comment, you can pass the prop here like this: `primary="true"`

Reply:
Alright